File: w09/d03/will_smith/mail_objects_app/app/assets/javascripts/package.js
function Package() {
  this.content = [];
  this.fragile = true;
  this.broken = false;
}

Package.prototype.shake = function(){
  if (this.fragile === true) {
    this.broken = true;
  }
};

Package.prototype.isBroken = function(){
  if (this.broken === true) {
    return 'yes, it is broken';
  } else {
    return 'no, it is no broken';
  }
};

File: w09/d01/matthew_korporaal/mail_object/package.rb
require_relative 'mail_item'

class Package < MailItem

  attr_accessor :content

  def initialize
    @content = []
    @fragile = false
    @broken = false
  end

  def fragile?
    @fragile
  end

  def broken?
    @broken
  end

  def shake
    @broken = true
  end

end